core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -120,6 +120,12 @@ object OptionT extends OptionTInstances {
     def apply[A](value: Option[A])(implicit F: Applicative[F]): OptionT[F, A] =
       OptionT(F.pure(value))
   }
+
+  /**
+    * Lifts the `F[A]` Functor into an `OptionT[F, A]`.
+    *
+    */
+  def liftF[F[_], A](fa: F[A])(implicit F: Functor[F]): OptionT[F, A] = OptionT(F.map(fa)(Option(_)))
```

Comment:
I used `Option(_)` to prevent `null` and getting `None` instead of `Some(null)` in that case, but if you confirm we should not prevent them from doing that I'll change it to `Some(_)`.